### ISSUE TITLE
aria-hidden display:none

### ISSUE THREAD

**Issue by thamas** [rvk-utd/hanna#1]
```css
[aria-hidden][aria-hidden="true"], [hidden][hidden] {
  display: none;
}
```
This is a bug! `aria-hidden="true"` is not for hiding sg from the page totally. It is for keeping it visible while hiding it from screen readers. 

Applying a `display: none` rule is a very bad idea IMHO and I do not understand what could be the reason for it!

**Comment by thamas** [comment]
@maranomynet ^^

**Comment by maranomynet** [comment]
I am so sorry. Totally missed the notifications for this.

Yeah, reserving `[aria-hidden]` for suppressing *visible* content is the more sensible way to go. This was a mistake.

However, explicitly normalizing `[hidden]` (double strong) to `display:none;` is a convenience styling to cut down on instances where `hidden` unexpectedly fails to visually hide elements because some low-specificity selectors set `display:flex` or `disply:inline-block`.

**Comment by maranomynet** [comment]
`[aria-hidden]` issue fixed in 4b759d50b4da95bc75acf3d10e1bf430f2cbfc38

**Comment by maranomynet** [comment]
This has now been published to the live [https://styles.reykjavik.is](https://styles.reykjavik.is/css/v0.8/-basics.css) server
